tests/components/rflink/test_sensor.py

```diff
@@ -180,10 +231,16 @@
     assert new_sensor
     assert new_sensor.state == "ok"
 
-    event_callback({"id": "test3", "sensor": "battery", "value": "ko", "unit": ""})
+    event_callback({
+        "id": "test3",
+        "sensor": "battery",
+        "value": "ko",
+        "unit": ""
+    })
     await hass.async_block_till_done()
     # tmp_entity must be deleted from EVENT_KEY_COMMAND
-    assert tmp_entity not in hass.data[DATA_ENTITY_LOOKUP][EVENT_KEY_SENSOR]["test3"]
+    assert tmp_entity not in hass.data[DATA_ENTITY_LOOKUP][EVENT_KEY_SENSOR][
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4679175)